lighter/callbacks/logger.py

```diff
@@ -0,0 +1,375 @@
+import re
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Union, Optional
+
+import torch
+import torch.distributed as dist
+import torchvision
+from loguru import logger
+from monai.utils.module import optional_import
+from pytorch_lightning import Callback, Trainer
+from torch.utils import tensorboard
+from yaml import safe_load
+
+from lighter import LighterSystem
+
+LIGHTNING_TO_LIGHTER_STAGE = {"train": "train", "validate": "val", "test": "test"}
+OPTIONAL_IMPORTS = {}
+
+class LighterLogger(Callback):
+
+    def __init__(self, project, log_dir, log_input_as=None, log_target_as=None, log_pred_as=None, tensorboard=False, wandb=False) -> None:
```

Comment:
Could we rename this to `log_input_as` to `input_type` and the others respectively? I think it `log` is unnecessary as its in the logger callback already

Reply:
Sounds good, will do